.gitignore

```diff
@@ -0,0 +1,11 @@
+# Compiled python modules.
+*.pyc
```

Comment:
updating to use `__pycache__/` but let me know if there's any other from this default list from github that you may want included and I can just toss them in: https://github.com/github/gitignore/blob/master/Python.gitignore